ceph.spec.in

```diff
@@ -1235,9 +1235,11 @@ This package provides Ceph default alerts for Prometheus.
 %autosetup -p1 -n @TARBALL_BASENAME@
 
 %build
-# LTO can be enabled as soon as the following GCC bug is fixed:
-# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48200
+# Disable lto on systems that do not support symver attribute
+# See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48200 for details
+%if 0%{?rhel} <= 8
```

Comment:
@smithfarm Can you please check what `suse_version` we should add here? I.e. from what suse version do you support the `symver` attribute?

Reply:
@b-ranto Please note that a bare `%if 0%{?rhel} <= 8` is bad because this will evaluate to true on lots of non-RH systems.

To fix that and also exclude openSUSE Leap 15.3 and SLE-15-SP3, we would need:

```
%if ( 0%{?rhel} && 0%{?rhel} <= 8 ) || ( 0%{?suse_version} && 0%{?suse_version} <= 1500 )
```

Reply:
@b-ranto And thanks for bringing me into the loop here :-) I do appreciate it.

Reply:
Ah, right, thanks for catching this. I have updated the conditional.